spec/features/publish/add_course_button_spec.rb

```diff
@@ -5,7 +5,6 @@
 feature 'Add course button', { can_edit_current_and_next_cycles: true } do
   scenario 'with no study sites on the provider in the next cycle and feature flag off' do
```

Comment:
` and feature flag off` should be removed

Reply:
Good catch! https://github.com/DFE-Digital/publish-teacher-training/pull/3816/commits/c0d1000f2bf9f98ba15ab3a5fd2933423b2f6594